src/templates/blog/post.js

```diff
@@ -49,16 +57,30 @@ BlogPostTemplate.propTypes = {
   html: PropTypes.string.isRequired,
   retinaCover: PropTypes.string,
   title: PropTypes.string.isRequired,
+  description: PropTypes.string,
 }
 
 export default ({ data }) => {
   const { markdownRemark } = data
   const { frontmatter, html } = markdownRemark
-  const { author, date, retina_cover: retinaCover, title } = frontmatter
+  const {
+    author,
+    date,
+    retina_cover: retinaCover,
+    title,
+    description,
```

Comment:
Do you mind keeping the alphabetic order? :grimacing: 

```suggestion
    author,
    date,
    description,
    retina_cover: retinaCover,
    title,
```